internal/manifests/collector/service.go

```diff
@@ -31,9 +31,7 @@ import (
 
 // headless and monitoring labels are to differentiate the headless/monitoring services from the clusterIP service.
 const (
-	headlessLabel   = "operator.opentelemetry.io/collector-headless-service"
```

Comment:
IMO we should keep these labels to not make a breaking change for people who rely on these

Reply:
Ok, I'll change it.